server/sonar-web/src/main/js/apps/quality-gates/components/Projects.js

```diff
@@ -19,54 +19,116 @@
  */
 import React from 'react';
 import escapeHtml from 'escape-html';
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'escapeHtml'. ([javascript:S1128](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1128))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3152&issues=AWIkUgyQ7DpOGbPQ2tAC&open=AWIkUgyQ7DpOGbPQ2tAC)

[ISSUE_KEY]: AWIkUgyQ7DpOGbPQ2tAC